cmd/commands/ceph.go

```diff
@@ -29,7 +29,7 @@ var CephCmd = &cobra.Command{
 	DisableFlagParsing: true,
 	Args:               cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		clientsets := GetClientsets(cmd.Context())
+		clientsets := GetClientsets(cmd.Context(), true)
```

Comment:
Could we separate this into two API calls such as this? To me, it would be more readable.
```suggestion
		clientsets := GetClientsets(cmd.Context())
		CheckOperatorPod(cmd.Context())
```

Reply:
agreed